dashboards-observability/public/framework/redux/store/index.ts

```diff
@@ -5,21 +5,45 @@
 
 import { configureStore } from '@reduxjs/toolkit';
 import rootReducer from '../reducers';
+import storage from 'redux-persist/lib/storage';
```

Comment:
should this use session storage instead of local storage?

Reply:
Yes, we should use session storage, the default storage is localStorage